test/common/stats/thread_local_store_test.cc

```diff
@@ -1955,7 +1990,62 @@ TEST_F(HistogramThreadTest, ScopeOverlap) {
   EXPECT_EQ(0, numTlsHistograms());
 
   shutdownThreading();
-  store_->histogramFromString("histogram_after_shutdown", Histogram::Unit::Unspecified);
+  scope_.histogramFromString("histogram_after_shutdown", Histogram::Unit::Unspecified);
+}
+
+class TestSinkPredicates : public Stats::SinkPredicates {
```

Comment:
@pradeepcrao I added this because coverage complained about ThreadLocalStore::setSinkPredicates not being covered. I realize this might conflict with your histogram predicates PR, but I don't think it will be hard to resolve. I feel like in that PR you might have broken out TestSinkPredicates into a header so we should clean this when that lands.

Reply:
SGTM.